clients/drcachesim/common/trace_entry.h

```diff
@@ -472,6 +476,16 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_MAYBE_BLOCKING_SYSCALL,
 
+    /**
+     * Indicates a point in the trace where a syscall's trace start.
```

Comment:
"syscall's kernel trace" would be more clear.  Ditto below.

Reply:
Done.